src/main/kotlin/three/consulting/epoc/service/TimesheetEntryService.kt

```diff
@@ -18,6 +19,9 @@ private val logger = KotlinLogging.logger {}
 class TimesheetEntryService(private val timesheetEntryRepository: TimesheetEntryRepository) {
 
     fun findTimesheetEntries(timesheetId: Long?, email: String?, startDate: LocalDate?, endDate: LocalDate?): List<TimesheetEntryDTO> {
+        if (!isAuthorizedToGetTimesheetEntries(email)) {
+            throw UnableToGetTimesheetEntriesException()
```

Comment:
This is sort of a controller level check done on service level, so you should throw an exception stating that the user is unauthorized or forbidden, see similar thing done in https://github.com/three-consulting/epoc-backend/blob/main/src/main/kotlin/three/consulting/epoc/service/TimesheetEntryService.kt#L86-L87